README.md

```diff
@@ -57,8 +55,7 @@ directory of GeoServer.
 
 ## Maven Integration
 
-GeoMesa artifacts are published to the LocationTech Maven repository. To include GeoMesa in your project,
-add the following repositories to your pom:
+GeoMesa artifacts are published to the [LocationTech Maven repository](https://repo.locationtech.org/content/groups/releases/org/locationtech/geomesa/). To include GeoMesa in your project, add the following repositories to your pom:
```

Comment:
@metasim These changes should be made to build/README.md as well.  That file is filtered during the build process to update version numbers, etc.